FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/DicomToDatalakeSchedulerService.cs

```diff
@@ -463,6 +463,7 @@ private async Task<CurrentTriggerEntity> CreateInitialTriggerEntity(Cancellation
                 TriggerSequenceId = 0,
                 StartOffset = InitialStartOffset,
                 EndOffset = latestSequence,
+                JobVersion = DicomToDatalakeJobVersionManager.CurrentJobVersion,
```

Comment:
we also need to set the job version for orchestrator job input data and updated trigger entity.